platzky/platzky.py

```
@@ -133,6 +138,7 @@ def page_not_found(e):
 
 def create_app_from_config(config: Config) -> Engine:
     engine = create_engine_from_config(config)
+    admin_blueprint = admin.create_admin_blueprint(login_methods=engine.login_methods, db=engine.db, locale_func=engine.get_locale)
```

Ruff (0.8.2)

Line too long (131 > 100)

(E501)